bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,98 @@
+from asyncio import TimeoutError
+from pathlib import Path
+from random import choice
+
+from discord import Color, Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+
+
+class Hangman(commands.Cog):
+    """
+    Cog for the Hangman game.
+
+    Hangman is a classic game where the user tries to guess a word, with a limited amount of tries.
+    """
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        with Path("bot/resources/fun/top_1000_used_words.txt").resolve().open(mode="r", encoding="utf-8") as file:
```

Comment:
`Path("bot/resources/fun/top_1000_used_words.txt")` could perhaps be stored in a global constant such as `TOP_WORDS_FILE_PATH`, to facilitate reading.